### DISCUSSION TITLE
Code signature changed?

### DISCUSSION THREAD

**Discussion by AmnaSN** [wavetermdev/waveterm#900] [General]
Has the code signature in your latest version for MacOS(https://dl.waveterm.dev/releases-w2/Wave-darwin-universal-0.8.5.dmg) changed from 'Michael Sawka (8M47V3YG4N)' to 'Command Line Inc (M4LA8V687Y)'?

**Comment by esimkowitz** [comment]
Hello, yes we switched from @sawka's personal developer account to an official corporate account with v0.8.0. It took us a while to get that processed, thanks Apple 😅
